.npmignore

```diff
@@ -0,0 +1 @@
+CHANGELOG.md
```

Comment:
With a `files` key, this is unnecessary.
However, I think `CHANGELOG.md` _should_ be included in `files`.

Reply:
I would've thought the same but I initially only added the files key, no npmignore, and changelog was included. I'm not sure what value chsngelogs have in built packages. Happy to have my mind changed though.

Reply:
Personally, I like having the changelog in the package as it saves me from going to the GitHub page to check it.

Reply:
> I would've thought the same but I initially only added the files key, no npmignore, and changelog was included.

You probably ran `yarn pack` which doesn't take `files` into account. `npm pack` is the correct command to run.

Reply:
Weird, I definitely ran `npm pack`. I wonder if changesets uses `npm pack` or not. I've removed the `.npmignore` for now.

Reply:
I made a few edits, but check the last link that confirms it uses `npm`.